wled00/wled00.ino

```diff
@@ -288,6 +285,7 @@ byte briOld = 0;
 byte briT = 0;
 byte briIT = 0;
 byte briLast = 128;                           //brightness before turned off. Used for toggle function
+byte whiteLast = 128;                         //white channel before turned off. Used for toggle function
```

Comment:
the last value of the white channel - this can be smitched ON/OFF with the remotes 3 and 4.